src/navier_stokes/INSVCStaggeredConservativeHierarchyIntegrator.cpp

```diff
@@ -708,9 +693,33 @@ INSVCStaggeredConservativeHierarchyIntegrator::integrateHierarchy(const double c
     // Store the density for later use
     d_hier_sc_data_ops->copyData(d_rho_linear_op_idx, d_rho_sc_scratch_idx, /*interior_only*/ true);
 
+    // Compute the Brinkman contribution to the velocity operator.
+    d_hier_sc_data_ops->setToScalar(d_velocity_L_idx, 0.0);
+    for (auto& brinkman_force : d_brinkman_force)
+    {
+        brinkman_force->demarcateBrinkmanZone(d_velocity_L_idx, new_time, cycle_num);
+    }
+
+    // Synchronize Brinkman coefficients.
+    SynchronizationTransactionComponent L_synch_transaction =
+        SynchronizationTransactionComponent(d_velocity_L_idx, "CONSERVATIVE_COARSEN");
+    d_side_synch_op->resetTransactionComponent(L_synch_transaction);
+    d_side_synch_op->synchronizeData(d_integrator_time);
+    d_side_synch_op->resetTransactionComponent(default_synch_transaction);
```

Comment:
It seems odd to me that we are using a completely different algorithm in this class now. Is this intentional?

Reply:
I didn't get that. What part of the algorithm looks completely different?

Reply:
Ah, its only optional: I understand now. I thought the Brinkman terms were mandatory and changed the behavior of this class.